admin-dev/themes/new-theme/scss/theme.scss

```diff
@@ -73,3 +73,8 @@ $color-icons: #6c868e;
   float:none;
   margin:0 auto;
 }
+
+.fullspace {
```

Comment:
Hey! Thank you!

As we try to follow the RSCSS norm, please add, for the helpers, an underscore : `_fullspace`.

http://rscss.io/

:+1: